Core/Program.cs

```diff
@@ -40,9 +40,11 @@ public static int Main (string[] args)
     {
       app.Run(args);
     }
-    catch (UserDoesNotWantToContinueException u)
+    catch (UserInteractionException e)
     {
-      Console.WriteLine(u.Message);
+      Console.WriteLine(e.Message);
+      var log = Log.ForContext(e.TargetSite!.DeclaringType);
+      log.Error(e, "User cancelled the operation or cannot continue with release due to inconsistencies in the repository or JIRA.");
```

Comment:
```suggestion
      log.Error(e, "User canceled the operation or cannot continue with release due to inconsistencies in the repository or JIRA.");
```

Reply:
Both versions are correct, and I really, REALLY prefer `cancelled`.

If this is about consistency though, I can take a step back ;)

Reply:
It's 'canceled' with one L for sourcecode and source code docs, because one-L is the version preferred in American English, which we use as style (https://www.grammarly.com/blog/canceled-vs-cancelled/). The one-L version in particular is also used more in .NET APIs bei Microsft (though no one is truly consistent here, including Microsoft and re-motion). So, the one-L variant is a recent (last couple of years) development and in order to be consistent we should strive towards this version. I, too, was for a long time confused and preferred the double-L version because it felt grammatically correct since '-ed' triggers two 'L' in the verb. Alas, real-word says nope so the more consistent we are, the easier it is for new committers to recognize the default case and follow the pattern :)

Reply:
OK, thank you for the in-depth explanation :)